storage/cloud-client/notification_polling.py

```diff
@@ -24,7 +24,11 @@
 https://cloud.google.com/storage/docs/reporting-changes or follow the steps
 below:
 
-1. Activate the Google Cloud Pub/Sub API, if you have not already done so.
+1. First, follow the common setup steps for these snippets, specically
+   configuring auth and installing dependencies. See the README's "Setup"
+   section.
+
+2. Activate the Google Cloud Pub/Sub API, if you have not already done so.
```

Comment:
Looks like you'll need to update all the numbers after this?

Reply:
Good catch, done.